python/cudf/cudf/core/series.py

```diff
@@ -1011,7 +1011,10 @@ def map(self, arg, na_action=None) -> "Series":
             result.name = self.name
             result.index = self.index
         else:
-            result = self.applymap(arg)
+            # TODO: switch to `apply`
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
```

Comment:
Can this be more specific?
```suggestion
                warnings.simplefilter("ignore::FutureWarning")
```